lib/ain-grpc/src/rpc/web3.rs

```diff
@@ -19,8 +19,9 @@ pub struct MetachainWeb3RPCModule {}
 
 impl MetachainWeb3RPCServer for MetachainWeb3RPCModule {
     fn client_version(&self) -> RpcResult<String> {
-        let version = env!("CARGO_PKG_VERSION");
-        let commit = option_env!("GIT_HASH").unwrap_or("unknown");
+        let version: String = ain_cpp_imports::get_client_version();
+        let commit = option_env!("GIT_HASH")
+            .ok_or_else(|| Error::Custom(format!("missing GIT_HASH env var")))?;
```

Comment:
No point in throwing a runtime error on something that fails during compile time.

```suggestion
        let commit = env!("GIT_HASH", ""missing GIT_HASH env var");
```

Reply:
as per @sieniven 's comment, i think we can expose the `GIT_HASH` via the ffi C++ bridge

Reply:
Resolved directly in PR #2107. web3_clientVersion RPC returns the format `Metachain/{client_version}/{target_os}-{target_arch}/{rustc_version}`.